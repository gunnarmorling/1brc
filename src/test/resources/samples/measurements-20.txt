Odesa1️⃣🐝🏎️;6.5
Canberra1️⃣🐝🏎️;5.2
Lhasa1️⃣🐝🏎️;13.4
Edinburgh1️⃣🐝🏎️;19.8
Da Nang1️⃣🐝🏎️;33.7
Xi'an1️⃣🐝🏎️;17.5
Berlin1️⃣🐝🏎️;-0.3
Tamanrasset1️⃣🐝🏎️;17.9
Abéché1️⃣🐝🏎️;27.3
Baghdad1️⃣🐝🏎️;26.0
Lyon1️⃣🐝🏎️;1.8
Mogadishu1️⃣🐝🏎️;11.5
Bangkok1️⃣🐝🏎️;25.6
Irkutsk1️⃣🐝🏎️;9.9
Parakou1️⃣🐝🏎️;36.3
Almaty1️⃣🐝🏎️;15.3
Birao1️⃣🐝🏎️;33.5
Chittagong1️⃣🐝🏎️;12.6
Tirana1️⃣🐝🏎️;27.7
Nashville1️⃣🐝🏎️;-4.9
