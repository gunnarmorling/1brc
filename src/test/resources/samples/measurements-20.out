{Abéché1️⃣🐝🏎️=27.3/27.3/27.3, Almaty1️⃣🐝🏎️=15.3/15.3/15.3, Baghdad1️⃣🐝🏎️=26.0/26.0/26.0, Bangkok1️⃣🐝🏎️=25.6/25.6/25.6, Berlin1️⃣🐝🏎️=-0.3/-0.3/-0.3, Birao1️⃣🐝🏎️=33.5/33.5/33.5, Canberra1️⃣🐝🏎️=5.2/5.2/5.2, Chittagong1️⃣🐝🏎️=12.6/12.6/12.6, Da Nang1️⃣🐝🏎️=33.7/33.7/33.7, Edinburgh1️⃣🐝🏎️=19.8/19.8/19.8, Irkutsk1️⃣🐝🏎️=9.9/9.9/9.9, Lhasa1️⃣🐝🏎️=13.4/13.4/13.4, Lyon1️⃣🐝🏎️=1.8/1.8/1.8, Mogadishu1️⃣🐝🏎️=11.5/11.5/11.5, Nashville1️⃣🐝🏎️=-4.9/-4.9/-4.9, Odesa1️⃣🐝🏎️=6.5/6.5/6.5, Parakou1️⃣🐝🏎️=36.3/36.3/36.3, Tamanrasset1️⃣🐝🏎️=17.9/17.9/17.9, Tirana1️⃣🐝🏎️=27.7/27.7/27.7, Xi'an1️⃣🐝🏎️=17.5/17.5/17.5}
